index.js

```diff
@@ -1,57 +1,79 @@
 'use strict'
 
-var level = require('level')
-var has = require('has')
-var pump = require('pump')
-var fs = require('fs')
-var net = require('net')
-var path = require('path')
-var multileveldown = require('multileveldown')
-
-module.exports = function (dir, opts) {
-  if (!opts) opts = {}
-  if (!has(opts, 'retry')) opts.retry = true
-
-  var sockPath = process.platform === 'win32'
+const level = require('level')
+const { pipeline: pump } = require('stream')
```

Comment:
In modules it's preferred to use `require('readable-stream')` to get a particular version. Or well, it used to be preferred, at a time where `stream` could mean streams1 or streams2. Still, considering the complexity of streams I'm leaning towards sticking with `require('readable-stream')` as a stable base. What do you think?

Reply:
We've fixed so many weird edge cases with streams in node 14 so I would really like to have those when possible... I'd be more inclined to use readable-stream when v4 is release.

Reply:
If we can't have those fixes for all the node versions that we support here, it's not worth it IMO. It's just as important that everyone has the same code base. When someone opens a bug report, I don't want to dive into multiple versions of `stream`.

Reply:
Fair enough.